src/Sulu/Bundle/DocumentManagerBundle/Resources/config/core.xml

```diff
@@ -89,7 +89,9 @@
             <argument type="service" id="sulu_document_manager.metadata_factory" />
         </service>
 
-        <service id="sulu_document_manager.proxy_manager.factory.ghost" class="ProxyManager\Factory\LazyLoadingGhostFactory" />
+        <service id="sulu_document_manager.proxy_manager.factory.ghost" class="Sulu\Component\DocumentManager\LazyLoadingGhostFactory">
+            <argument>%sulu.cache_dir%/documents</argument>
```

Comment:
A better way would be to create the directory in a cache warmer and instantiate the Configuration here and set the cache directory...